test/basic.csr.test.js

```diff
@@ -143,6 +143,14 @@ test('/redirect2', async t => {
   t.is(await page.$text('h1'), 'Index page')
 })
 
+test('/redirect3', async t => {
+  // New page for redirecting to external link.
+  const page = await browser.page(url('/'))
+  await page.nuxt.navigate('/redirect3', false)
+  await page.waitForFunction(() => window.location.href === 'https://nuxtjs.org/')
+  t.pass()
```

Comment:
Don't forget to use `page.close()` here since you create a new instance :)

Reply:
I'm trying to replace `url` and will change the test.
About the `ctx.app.error`, I didn't find an appropriate way to jump out of the rendering except `ctx.app.error` 😓